<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Config QtQuickTerm</string>
  </property>
  <property name="windowIcon">
   <iconset theme="configure"/>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>291</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>290</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Font:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFontComboBox" name="fontComboBox">
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string notr="true">Andale Mono</string>
      </property>
      <property name="fontFilters">
       <set>QFontComboBox::MonospacedFonts|QFontComboBox::NonScalableFonts|QFontComboBox::ScalableFonts</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Color scheme:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="schemeComboBox">
      <property name="toolTip">
       <string/>
      </property>
      <property name="currentText">
       <string notr="true">BlackOnLightYellow</string>
      </property>
      <item>
       <property name="text">
        <string notr="true">BlackOnLightYellow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string notr="true">BlackOnRandomLight</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string notr="true">BlackOnWhite</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string notr="true">GreenOnBlack</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string notr="true">Linux</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string notr="true">Solarized</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string notr="true">SolarizedLight</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string notr="true">WhiteOnBlack</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="fontSize">
      <property name="minimum">
       <number>10</number>
      </property>
      <property name="maximum">
       <number>30</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Font size:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>89</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Preview:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="fontPreview">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string>Sample text</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>159</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Width:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSpinBox" name="widthSpinBox">
      <property name="minimum">
       <number>400</number>
      </property>
      <property name="maximum">
       <number>1600</number>
      </property>
      <property name="singleStep">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="heightSpinBox">
      <property name="minimum">
       <number>300</number>
      </property>
      <property name="maximum">
       <number>1080</number>
      </property>
      <property name="singleStep">
       <number>50</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
